chart/values.yaml

```
@@ -349,6 +349,8 @@ csi:
       ctrl_loss_tmo: "1980"
       # Kato (keep alive timeout) in seconds
       keep_alive_tmo: ""
+      # Fallback to nvme connect over tcp if host is not rdma capable
```

Maybe something like this?

```suggestion
      # -- Fallback to nvme-tcp if nvme-rdma is enabled but not available on a particular csi-node
```

---

done